pylint/checkers/variables.py

```diff
@@ -992,6 +992,11 @@ def visit_assignname(self, node: nodes.AssignName) -> None:
     def visit_delname(self, node: nodes.DelName) -> None:
         self.visit_name(node)
 
+    @utils.check_messages(
+        "undefined-variable",
+        "used-before-assignment",
+        "cell-var-from-loop",
+    )
```

Comment:
@cdce8p @Pierre-Sassoulas This causes a regression as the `unused_import_everything_disabled` functional test no longer passes. I'm not sure what's going on here, but this is crucial apparently.